utils/mobility/status.ts

```diff
@@ -29,11 +29,11 @@ export const checkStatus = (data: any, msgIdSet: any) => {
     Object.assign(errorObj, contextRes.ERRORS)
   }
 
-  if (!message.ref_id) {
-    const key = `${mobilitySequence.STATUS}_ref_id`
-    errorObj[key] = `ref_id in /${constants.STATUS} must be present`
+  if (!message?.ref_id && !message?.order_id) {
+    const key = `${mobilitySequence.STATUS}_id`
+    errorObj[key] = `either of ref_id or order_id must be present in /${constants.STATUS}`
```

Comment:
is this allowed?

Reply:
If a buyer wants to fetch the latest status for a transactional journey he can do so by sending either of ref_id(order not created) or order_id(order created)